tests/acceptance/02_classes/02_functions/usemodule-returns-nonzero.cf

```diff
@@ -0,0 +1,110 @@
+#!/var/cfengine/bin/cf-agent -f-
+body common control
+{
+        inputs => { "../../default.cf.sub" };
+        bundlesequence  => { default("$(this.promise_filename)") };
+        version => "1.0";
+}
+bundle agent init
+{
+  vars:
+      "foo_usemodule" string =>
+"#!/bin/sh
+echo '=mystring=foo99'
+echo '+class_from_classes_usemodule'
+exit 1
+";
+
+      "foo_commands_module" string =>
+      "#!/bin/sh
+echo '=mystring=foo99'
```

Comment:
indent this shebang back to `^` (start of line)?

Reply:
sure i can,